packages/page-settings/src/Metadata/Extensions.tsx

```diff
@@ -37,14 +49,16 @@ function Extensions ({ chainInfo, className }: Props): React.ReactElement<Props>
       if (chainInfo && extensions?.[selectedIndex]) {
         toggleBusy();
 
+        const rawDef: MetadataDef = objectSpread<MetadataDef>({}, { ...chainInfo, rawMetadata: rawMetadata || '0x00' });
```

Comment:
I think it can just be `rawMetadata` can just be `rawMetadata || undefined`

Since we now assert in the extension.